engine.json

```diff
@@ -6,7 +6,12 @@
     "params" : {
       "name": "sample-handmade-data.txt",
       "appName": "handmade",
-      "eventNames": ["purchase"]
+      "eventNames": ["purchase"],
+      "eventWindow": {
+        "duration": "1 day", // default is to not drop events
+        "removeDuplicates": "true",  // default = false
+        "compressProperties": "true" // default = false
+      }
     }
```

Comment:
? non-standard json ? does this actually work?

best use `"comment": "some textual comment" or better yet edit the README.md to document. I can do the README.md if you want.

We should create a secondary engine.json like we did for the other integration test. Then copy the user's to a safe location, copy the test engine to engine.json and after the test restore to user-engine.json. See examples/integration-test